build/leafdoc-templates/html.hbs

```diff
@@ -143,7 +143,10 @@
 
 
 	<div class="footer">
-		<p>© 2017 <a href="http://agafonkin.com/en">Vladimir Agafonkin</a>. Maps © <a 	href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors.</p>
+		<p>
+			© 2010–2018 <a href="http://agafonkin.com/en">Vladimir Agafonkin</a>.
```

Comment:
This should be 2019 as well, right?

Reply:
Sure, thanks for spotting this glitch (2018+1=2019, thus 2017+1=2018).